tcms/search/__init__.py

```diff
@@ -44,10 +44,9 @@ def advance_search(request):
         product_choice = []
         for product in Product.objects.all():
             product_choice.append((product.pk, product.name))
-        plan_type_choices = PlanType.objects.all()  # pylint: disable=unused-variable
```

Comment:
This two are used. On line 50 we call `render(request, 'search/advanced_search.html', locals())`. `locals()` returns all the local variables, including the ones you deleted. They are later used to render the response. You may have a failing test about this.